spec/modeltr/document_spec.py

```diff
@@ -0,0 +1,376 @@
+from mamba import description, context, it
```

Comment:
Great to start with the test, we need even more of them - will be added as separate commit.